packages/mail/script/sendMailsToSegment/cleanedUserSubscriptionInvitation.js

```diff
@@ -0,0 +1,77 @@
+const yargs = require('yargs')
+const PgDb = require('@orbiting/backend-modules-base/lib/PgDb')
+const dayjs = require('dayjs')
+
+const sendMailsToSegment = require('./sendMailsToSegment')
+
+const argv = yargs
+  .option('dry-run', {
+    default: true,
+  })
+  .option('from', {
+    coerce: dayjs,
+    default: dayjs().subtract(30, 'day'),
+  })
+  .option('to', {
+    coerce: dayjs,
+    default: dayjs(),
+  })
+  .help()
+  .version().argv
+
+PgDb.connect().then(async (pgdb) => {
+  if (argv.dryRun) {
+    console.warn('In dry-run mode. Use --no-dry-run to send emails to segment.')
+  }
+
+  console.log(dayjs(argv.to).format('YYYY-MM-DD'))
+  console.log(dayjs(argv.from).format('YYYY-MM-DD'))
+  const mail = {
+    subject: 'Warum Sie von uns keine Newsletter mehr erhalten',
+    templateName: 'cleaned_user_subscription_invitation',
+  }
+
+  const emailAddresses = await pgdb.queryOneColumn(
+    `
+    WITH records AS (
+      WITH data AS (
+        SELECT
+          DISTINCT ON ("email")
+          "email",
+          "createdAt",
```

Comment:
Found a flaw in original query: Better to query `firedAt` throughout instead of `createdAt`. Latter is time of insertion into database, and that might lag behind if webhook flow is broken.